File: algorithm/python/BOJ2475.py
# problem : https://www.acmicpc.net/problem/2475

import sys

koi_id = list(map(int, sys.stdin.readline().rstrip().split()))

total = 0
for id in koi_id:
    total += pow(id, 2)

answer = total % 10
print(answer)

File: algorithm/typescript/BOJ2475.ts
// problem : https://www.acmicpc.net/problem/2475

import fs = require("fs");

const koiId: number[] = fs.readFileSync("/dev/stdin").toString().split(" ").map(Number);

let sum: number = 0;
koiId.forEach((num) => {
  sum += Math.pow(num, 2);
});

let answer: number = sum % 10;
console.log(answer);
